topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -60,10 +70,21 @@ recordings:
   youtube_id: vnFQ2fR_fzw
   speakers:
   - wm75
+- youtube_id: -HQB1zyWWXM
+  length: 55M
+  galaxy_version: 23.0 + 24.1 (where the UI differences matter most)
```

Comment:
@shiltemann: Is this fine to indicate a mix of Galaxy versions like this? Or is there a special form needed for postprocessing?

Reply:
@teresa-m please just take the most recent, 24.1 here